code/game/gamemodes/meteor/meteors.dm

```diff
@@ -319,31 +356,102 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 	if(!isspaceturf(T))
 		new /obj/effect/decal/cleanable/xenoblood(T)
 
-//Station buster Tunguska
-/obj/effect/meteor/tunguska
-	name = "tunguska meteor"
-	icon_state = "flaming"
-	desc = "Your life briefly passes before your eyes the moment you lay them on this monstrosity."
-	hits = 30
+//Blobby-Meaty Ore
+/obj/effect/meteor/meaty/mouldy
+	name = "mouldy ore"
+	desc = "Ewww."
+	color = "#654321"
+	meteorsound = 'sound/effects/splat.ogg'
+	meteordrop = list(/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/zombie, /obj/item/organ/tongue/zombie)
+	threat = 10
+	var/spawn_max = 4
+	var/blobbo_chance = 25
+
+/obj/effect/meteor/meaty/mouldy/meteor_effect()
+	..()
+	if(prob(blobbo_chance))
+		new /mob/living/simple_animal/hostile/blob/blobbernaut/independent(src.loc, src)
+	else
+		for(var/i in 1 to spawn_max)
+			var/mob/living/simple_animal/hostile/blob/blobspore/BS = new /mob/living/simple_animal/hostile/blob/blobspore(src.loc, src)
+			for(var/j = 1, j <= rand(1, 3), j++)
+				step(BS, pick(NORTH,SOUTH,EAST,WEST))
+
+//Bone-Meaty Ore
+/obj/effect/meteor/meaty/bone
+	name = "boney ore"
+	desc = "This thing sure drank its calcium."
+	icon = 'icons/obj/statuelarge.dmi'
+	icon_state = "skull"
+	meteordrop = list(/obj/item/stack/sheet/bone, /obj/item/organ/tongue/bone, /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/skeleton)
+	dropamt = 8 //No bone gib spawner, so lots of bones instead. Besides, bones would survive impact better than meat.
+	hits = 4 // Bone is pretty hard.
+	meteorsound = 'sound/magic/RATTLEMEBONES2.ogg' // ME BONES
+	threat = 5
+	
+/obj/effect/meteor/meaty/bone/make_debris()
+	for(var/throws = dropamt, throws > 0, throws--)
+		var/thing_to_spawn = pick(meteordrop)
+		var/atom/thing = new thing_to_spawn(get_turf(src))
+		for(var/j = 1, j <= rand(1, 3), j++)
+			step(thing, pick(NORTH,SOUTH,EAST,WEST))
+
+//Long ago in a distant fairy land, there was a man known far and wide...
+/obj/effect/meteor/meaty/meatball_man
+	name = "meatball man"
+	icon = 'icons/effects/512x512.dmi'
+	icon_state = "meatball_man"
+	pixel_x = -240
+	pixel_y = -240
+	desc = "Oh, what a feat! He's one with the meat!"
+	hits = INFINITY // You cannot stop the Meatball Man.
+	lifetime = 0 // Meatball Man is eternal.
+	spin = FALSE
 	hitpwr = 1
 	heavy = 1
-	meteorsound = 'sound/effects/bamf.ogg'
-	meteordrop = list(/obj/item/stack/ore/plasma)
-	threat = 50
-
-/obj/effect/meteor/tunguska/Move()
+	meteorsound = 'sound/effects/blobattack.ogg'
+	threat = 200
+	move_delay = 40 // Meatball Man never rushes, for all will become Meat in time.
+	var/meaty_range = 14
+	var/eaty_range = 7
+	var/meat_spreadiness = 20
+	var/he_is_close = FALSE
+
+/obj/effect/meteor/meaty/meatball_man/Move()
+	. = ..()
+	for(var/tile in spiral_range_turfs(eaty_range, src))
+		var/turf/T = tile
+		if(!T || !isturf(loc))
+			continue
+		T.ex_act(hitpwr)
+		for(var/atom/thing in T)
+			if(isturf(loc) && thing != src)
+				thing.ex_act(hitpwr)
```

Comment:
would this ex_act() a mob standing on a tile twice (once because the turf got ex_act()ed, and once because the mob itself got ex_act()ed), or do ex_act()s on turfs not ex_act() everything in that turf?

Reply:
I don't know, but if you're standing directly in it's path you are going to get ex_acted a lot anyway because it does it every move. I'd advise moving when you see meat appear around you.

Reply:
Also this is pretty much how the singularity does it anyway.